docs/how_to/sessionize.md

```diff
@@ -2,73 +2,78 @@
 
-# Read files into table expressions with ibis.read:
-data = ibis.read("https://storage.googleapis.com/ibis-tutorial-data/wowah_data/wowah_data_raw.parquet")
+# Read files into table expressions with ibis.read_parquet:
+data = ibis.read_parquet(
+    "https://storage.googleapis.com/ibis-tutorial-data/wowah_data/wowah_data_raw.parquet"
+)
 
-# integer delay in seconds noting if a row should be included in the previous session for an entity
+# Integer delay in seconds noting if a row should be included in the previous
+# session for an entity.
 session_boundary_threshold = 30 * 60
 
 # Window for finding session ids per character
 entity_window = ibis.cumulative_window(group_by=c.char, order_by=c.timestamp)
 
 # Take the previous timestamp within a window (by character ordered by timestamp):
-# Note: the first value in a window will be null
+# Note: the first value in a window will be null.
 ts_lag = c.timestamp.lag().over(entity_window)
 
-# Subtract the lag from the current timestamp to get a timedelta
+# Subtract the lag from the current timestamp to get a timedelta.
 ts_delta = c.timestamp - ts_lag
 
 # Compare timedelta to our session delay in seconds to determine if the
 # current timestamp falls outside of the session.
-# Cast as int for aggregation
+# Cast as int for aggregation.
 is_new_session = (ts_delta > ibis.interval(seconds=session_boundary_threshold))
 
-# Window for finding session min/max
+# Window to compute session min/max and duration.
 session_window = ibis.window(group_by=[c.char, c.session_id])
 
 # Generate all of the data we need to analyze sessions:
 sessionized = (
     data
     # Create a session id for each character by using a cumulative sum
-    # over the `new_session` column
+    # over the `new_session` column.
     .mutate(new_session=is_new_session.fillna(True))
     # Create a session id for each character by using a cumulative sum
-    # over the `new_session` column
+    # over the `new_session` column.
     .mutate(session_id=c.new_session.sum().over(entity_window))
-    # Drop `new_session` because it is no longer needed
+    # Drop `new_session` because it is no longer needed.
     .drop("new_session")
     .mutate(
-        # Get session duration using max(timestamp) - min(timestamp) over our window
+        # Get session duration using max(timestamp) - min(timestamp) over our window.
         session_duration=c.timestamp.max().over(session_window) - c.timestamp.min().over(session_window)
     )
-    # Sort for convenience
+    # Sort for convenience.
     .order_by([c.char, c.timestamp])
 )
 ```
+
+Calling `ibis.show_sql(sessionized)` displays the nested SQL query and can be used to confirm that this Ibis expression does not rely on any join operation.
+
+Calling `sessionized.execute()` should complete in less than a minute, depending on the speed of the internet connection to download the data and the number of CPU cores available to parallelize the processing of this nested query.
```

Comment:
Hopefully no one comes us to saying that things are slow with a tiny single-core machine and a dialup connection 😂